zsh/configs/post/path.zsh

```diff
@@ -1,9 +1,11 @@
 # ensure dotfiles bin directory is loaded first
 PATH="$HOME/.bin:/usr/local/sbin:$PATH"
 
-# load ASDF, falling back to rbenv if not available
-if [ -d "$HOME/.asdf" ]; then
+# load ASDF, either from home dir or brew install
+if [ -f "$HOME/.asdf/asdf.sh" ]; then
   . $HOME/.asdf/asdf.sh
+elif ASDF_PREFIX=$(brew --prefix asdf); then
```

Comment:
I'm not great with shell scripting, but I presume this checks if the directory exists? When I run `brew --prefix asdf` on my machine where asdf was not installed via brew, it still returns `/usr/local/Cellar/asdf/0.7.1` even though that file is not there. Does the `elif` return false if the variable assignment fails? I did notice that running `$(brew --prefix asdf)` does give an error (and a 127 exit), so I presume that is falsey.

Reply:
Oh, interesting. I assumed this command would not return anything unless `asdf` was actually installed. I'm gonna have to rethink this.